docs/core/logger.md

```diff
@@ -567,17 +567,24 @@ You can change the order of [standard Logger keys](#standard-structured-keys) or
     --8<-- "examples/logger/src/reordering_log_keys_output.json"
     ```
 
-#### Setting timestamp to UTC
+#### Setting timestamp to custom Timezone
 
-By default, this Logger and standard logging library emits records using local time timestamp. You can override this behavior via `utc` parameter:
+By default, this Logger and standard logging library emits records using default lambda time(UTC) timestamp. You can enforce this behavior via `utc` parameter.
```

Comment:
```suggestion
By default, this Logger and the standard logging library emit records with the default AWS Lambda timestamp in UTC.
```

I don't think we need to mention the enforce, this is the default behavior. To be honest, I think there is no use for the `utc` field anymore.